test/test_mps.py

```diff
@@ -8911,29 +8911,40 @@ def test_lstm_1(self, device="mps", dtype=torch.float32):
         self.assertEqual(cpu_hn, hn)
         self.assertEqual(cpu_cn, cn)
 
-    @unittest.skipIf(True, "Backward of lstm returns wrong result")
-    def test_lstm_2(self, device="mps", dtype=torch.float32):
+    def test_lstm_backward_one_layer(self, device="mps", dtype=torch.float32):
+        layers = 1
```

Comment:
For future test with several layers. Now, it passes.

But fails with two layers because of incorrect forward